code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -95,6 +95,23 @@
 	mag_display_ammo = TRUE
 	empty_indicator = TRUE
 
+/obj/item/gun/ballistic/automatic/plastikov
```

Comment:
if the value is the same as 

/obj/item/gun/ballistic/automatic

you don't need to set it.